build.gradle.kts

```diff
@@ -21,16 +20,16 @@ repositories {
 }
 
 dependencies {
-	compileOnlyApi("com.destroystokyo.paper:paper-api:1.16.5-R0.1-SNAPSHOT")
+	compileOnlyApi("io.papermc.paper:paper-api:1.17-R0.1-SNAPSHOT")
 	compileOnly("org.jetbrains:annotations:20.1.0")
-	compileOnlyApi("com.intellectualsites.fawe:FAWE-Bukkit:1.16-637")
+	compileOnlyApi("com.fastasyncworldedit:FAWE-Bukkit:1.17-23")
```

Comment:
1.17-44 & use https://mvn.intellectualsites.com/content/groups/public/ as maven repository replacing the current two repositories from us.